data/json/mutations.json

```diff
@@ -3926,10 +3926,7 @@
         "cancels" : ["VEGETARIAN", "HERBIVORE", "RUMINANT",  "GRAZER"],
         "leads_to" : ["SAPROVORE"],
         "category" : ["MUTCAT_LIZARD", "MUTCAT_BEAST", "MUTCAT_SPIDER", "MUTCAT_CHIMERA", "MUTCAT_RAPTOR", "MUTCAT_FELINE"],
-        "vitamin_rates": [
-            [ "VitA", -999 ],
-            [ "VitC", -999 ]
-        ]
+        "vitamin_rates": [ [ "vitC", -2 ] ]
```

Comment:
This either doesn't work or doesn't work well enough. My carnivore character keeps losing vitC at a rate very similar to other vitamins.